e2e/e2e_test.go

```diff
@@ -139,8 +139,8 @@ func TestCreatesSecureCluster(t *testing.T) {
 			requireDatabaseToFunction(t, sb, builder)
 
 			//state, err := sb.Diff()
-			_, err := sb.Diff()
-			require.NoError(t, err)
+			//_, err := sb.Diff()
```

Comment:
This is not going to work anymore without a major refactor.

Reply:
@keith-mcclellan because we have a job now the e2e tests are not comparing exported yaml.  Personally, I do not like this type of testing, because it is very fragile.  Not certain we want to fix it.  But it is worth a conversation.